ws-services/src/main/java/org/egov/waterconnection/repository/builder/WsQueryBuilder.java

```diff
@@ -185,4 +216,12 @@ private String addPaginationWrapper(String query, List<Object> preparedStmtList,
 		preparedStmtList.add(limit + offset);
 		return PAGINATION_WRAPPER.replace("{}",query);
 	}
+	
+	private void addORClauseIfRequired(List<Object> values, StringBuilder queryString){
```

Comment:
what is this being used for?

Reply:
This function is use to append "WHERE"  or "OR" clause in sql query 
 Because of mobile search criteria  we are appending property Id and connection holder user id.
So to decided whether to use "WHERE" clause or 'OR" clause in search query we are using this function